yt/fields/particle_fields.py

```diff
@@ -803,54 +803,21 @@ def _pfunc_avg(field, data):
                        units = field_units)
     return fn
 
+DEP_MSG_SMOOTH_FIELD = (
+    "The notion of a smoothed field no longer exists since yt-4.0, because the "
+    "global octree is removed. Now the SPH particle fields directly accessible "
+    "from disk are treated properly in the visualization machinaries. So there "
+    "is no need to add a separate smoothed field. "
+    "See this page to learn more:\n"
+    "https://yt-project.org/doc/yt4differences.html"
+)
```

Comment:
@ngoldbaum Does the warning message look good to you now?

Reply:
I think it's true to say that yt doesn't have smoothed fields at all, since SPH smoothing onto images, octrees, and grids is still a thing. Maybe just say that it's no longer necessary to add a field specifically for smoothing, as a particle field will be smoothed onto a data object as appropriate and field values will be returned at particle locations otherwise.

Reply:
Thanks for the feedback. How about this:

Since yt-4.0, it's no longer necessary to add a field specifically for smoothing, because the global octree is removed. The old behavior of interpolating onto a grid structure can be recovered through data objects like `ds.arbitrary_grid`, `ds.covering_grid`, and most closely `ds.octree`. The visualization machinery now treats SPH fields properly by smoothing onto pixel locations. See this page to learn more: https://yt-project.org/doc/yt4differences.html

Reply:
That's perfect :)